hacking/aws_config/testing_policies/ses-policy.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowSesUsage",
+      "Effect": "Allow",
+      "Action": [
+        "ses:DeleteIdentity",
```

Comment:
For some reasons the rest of the SES permissions are in the [compute-policy.json](https://github.com/ansible/ansible/blob/devel/hacking/aws_config/testing_policies/compute-policy.json#L227)

I like the idea of pulling the ses permissions out into it's own policy file since it's a bit of a different area if we're going to do this it would be good to move the rest of the SES permissions into ses-policy.json as well.

I've asked on the #ansible-aws irc channel if there's a reason to keep these in compute instead.

Reply:
We can't have more than 10 policies (that's the most policies you can attach to a single group), so no new policies please (if you do want a new policy, have a look at combining existing policies first) - unless you want to add another group

Reply:
Makes sense. Given that best to move these permissions into compute-policy.json with the rest of the SES permissions.